src/Illuminate/Testing/TestResponse.php

```diff
@@ -1153,10 +1152,10 @@ public function assertSessionMissing($key)
     /**
      * Assert that the response is a given resource.
      *
-     * @param JsonResource $resource
+     * @param \Illuminate\Http\Resources\Json\JsonResource $resource
      * @return $this
      */
-    public function assertResource(JsonResource $resource)
+    public function assertResource(\Illuminate\Http\Resources\Json\JsonResource $resource)
```

Comment:
Please don't use fully qualified names in the actual type here. Only in the phpdoc, like the rest of the code.